### ISSUE TITLE
webview throws errors when remote module is disabled and plugins are installed

### ISSUE THREAD

**Issue by jdoubleu** [electron/electron#17021]
* **Electron Version** (output of `node_modules/.bin/electron --version`):
  * `4.0.1`
* **Operating System** (Platform and Version):
  * `macOS 10.14.3`
* **Last known working Electron version** (if applicable):
  * `n/a`

### Expected Behavior
There are no errors, whatsoever, when [disabling remote module](https://electronjs.org/docs/api/webview-tag#enableremotemodule) in `<webview>` tag.

### Actual behavior
The script `/path/to/project/node_modules/electron/dist/Electron.app/Contents/Resources/electron.asar/renderer/extensions/storage.js` throws:
```
Uncaught TypeError: Cannot destructure property `app` of 'undefined' or 'null'.
    at Object.<anonymous> (path/to/project/node_modules/electron/dist/Electron.app/Contents/Resources/electron.asar/renderer/extensions/storage.js:6)
    at Object.<anonymous> (path/to/project/node_modules/electron/dist/Electron.app/Contents/Resources/electron.asar/renderer/extensions/storage.js:140)
    at Module._compile (internal/modules/cjs/loader.js:693)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:704)
    at Module.load (internal/modules/cjs/loader.js:602)
    at tryModuleLoad (internal/modules/cjs/loader.js:541)
    at Function.Module._load (internal/modules/cjs/loader.js:533)
    at Module.require (internal/modules/cjs/loader.js:640)
    at require (internal/modules/cjs/helpers.js:20)
    at Object.exports.injectTo (path/to/project/node_modules/electron/dist/Electron.app/Contents/Resources/electron.asar/renderer/chrome-api.js:180)
```

### To Reproduce
```sh
$ git clone https://github.com/jdoubleu/electron-bug-17021.git
$ npm install
$ npm run start
```

Essentially it does the following things:
1. Installs a Chrome Plugin (i.e. Vue.js devtools)
2. Inlcudes a `<webview>` tag in the app with the `enableremotemodule="false"` attribute.
2. Opens the DevTools of the webview (e.g. `webview.addEventListener('dom-ready', event => event.target.openDevTools())`)

### Screenshots
![screenshot 2019-02-18 at 11 07 47](https://user-images.githubusercontent.com/11274444/52943441-72ae7b00-336d-11e9-871a-f817c3c7565d.png)

### Additional Information
It looks like the `storage.js` script requires access to the remote module to the the app's `userData` path. This throws an error when the remote module is disabled.

It looks like this feature has been introduced in https://github.com/electron/electron/pull/9951 (https://github.com/electron/electron/pull/9951/files#diff-07041300f132f9b519eeb86b31818b7e).

This issue only arises if the remote module is disabled AND a chrome plugin is installed. I haven't tested it with other plugins, yet.
The `content-scripts-injector.js` actually starts the stack trace seen above: https://github.com/electron/electron/blob/master/lib/renderer/content-scripts-injector.js#L18.

**Comment by miniak** [comment]
This is only fixed in Electron 6 beta. We would need to backport these changes:
- https://github.com/electron/electron/pull/16607
- https://github.com/electron/electron/pull/16739
- https://github.com/electron/electron/pull/16686

cc @MarshallOfSound

**Comment by jdoubleu** [comment]
Closing because it is fixed in the latest electron version (`6.0.4`).
